config/routes.rb

```diff
@@ -14,6 +17,10 @@
 
   resources :users do
     resources :tracks, as: :stems
+    collection do
+      get 'new_sc'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.